main_routers/live2d_router.py

```
@@ -730,6 +775,29 @@ def get_model_files_by_id(model_id: str):
         if not os.path.exists(model_dir):
             logger.warning(f"模型目录不存在: {model_dir}")
             return {"success": False, "error": "模型不存在"}
+
+        # 定位模型配置文件（支持 item 根目录或一级子目录）
```

Ruff (0.15.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)